src/main/webapp/js/components/csv/csv-config-page.jsx

```diff
@@ -164,6 +165,13 @@ class CsvConfigPage extends Component {
               <FieldArray name="csvFields">
                 { ({ fields }) => (
                   <div>
+                    {fields.length === 0 ? _.map(requiredFieldConfigIds, elem => fields.push({
```

Comment:
you should add those fields as initial value for the form instead of pushing it to the field during render, it should be done in the CsvConfigCreate component in the onChange method for the entity